opm/parser/eclipse/Deck/DeckRecord.cpp

```diff
@@ -29,6 +29,10 @@
 namespace Opm {
 
 
+    DeckRecord::DeckRecord( size_t size ) {
```

Comment:
It's pretty significant actually, I seem to recall 8-10% improvement. Mostly because reallocating here is expensive because it invokes clone.